msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/User-Specific-Views.md

```diff
@@ -215,12 +216,16 @@ const cardRes = {
 
 Card design guidelines to keep in mind while designing User Specific Views:
 
-* You can create a maximum of **60 User Specific Views** for a particular card sent to a chat or channel by specifying their `userIds` in the `refresh` section.
-* **Base Card:** The base version of the card that the bot developer sends to the chat. The base version is the version of the Adaptive Card for all users who are not specified in the `userIds` section.
+* **Refresh Behaviour:** You can create a maximum of **60 User Specific Views** for a particular card sent to a chat or channel by specifying their `userIds` in the `refresh` section. Alternatively, you may choose to skip `userIds` in the refresh property altogether in case the scenario involves <=60 users in Teams group chats/ channels. The Teams client will automatically invoke refresh calls for all the users if the group/channel has <=60 users.
```

Comment:
```suggestion
* **Refresh Behaviour:** You can create a maximum of **60 User Specific Views** for a particular card sent to a chat or channel by specifying their `userIds` in the `refresh` section. Alternatively, you may choose to skip `userIds` in the refresh property altogether in case the scenario involves <=60 users in Teams group chats/ channels. The Teams client will automatically invoke refresh calls for all the users if the group/channel has <=60 users.  If the `userIds` list property is specified as `userIds: []` in the refresh section of the card, the card is not automatically refreshed. Instead, a Refresh Card option is displayed to the user in the triple dot menu in web or desktop and in the long press context menu in mobile to manually refresh the card.
```

Use bullets for different lines